src/assets/mocks/achievement/achievement-list.json

```diff
@@ -0,0 +1,62 @@
+[
+    {
+      "id": 1,
+      "title": "Regional Branding Liaison",
+      "description": "card program",
+      "message": "Try to compress the AGP driver, maybe it will copy the multi-byte protocol!"
+    },
+    {
+      "id": 2,
+      "title": "Human Quality Administrator",
+      "description": "Clothing",
+      "message": "You can't override the capacitor without synthesizing the mobile XSS sensor!"
+    },
+    {
+      "id": 3,
+      "title": "Internal Creative Technician",
+      "description": "repurpose Fantastic calculating",
+      "message": "Use the primary RSS capacitor, then you can generate the back-end port!"
+    },
+    {
+      "id": 4,
+      "title": "Dynamic Functionality Liaison",
+      "description": "Money Market Account",
+      "message": "I'll copy the neural HDD driver, that should monitor the HDD monitor!"
+    },
+    {
+      "id": 5,
+      "title": "Direct Accountability Coordinator",
+      "description": "parsing transmit upward-trending",
+      "message": "calculating the protocol won't do anything, we need to copy the virtual SSL system!"
+    },
+    {
+      "id": 6,
+      "title": "Dynamic Program Executive",
+      "description": "Specialist withdrawal Ergonomic Steel Mouse",
+      "message": "Use the 1080p GB transmitter, then you can navigate the redundant protocol!"
+    },
+    {
+      "id": 7,
+      "title": "Lead Mobility Architect",
+      "description": "Sausages Cape Verde Escudo",
+      "message": "You can't input the transmitter without generating the bluetooth FTP firewall!"
+    },
+    {
+      "id": 8,
+      "title": "Forward Applications Developer",
+      "description": "Bedfordshire Swedish Krona mobile",
+      "message": "If we copy the application, we can get to the AI hard drive through the back-end CSS pixel!"
+    },
+    {
+      "id": 9,
+      "title": "Customer Mobility Designer",
+      "description": "matrix",
+      "message": "You can't hack the protocol without indexing the 1080p XSS driver!"
+    },
+    {
+      "id": 10,
+      "title": "Dynamic Configuration Supervisor",
+      "description": "payment Keyboard tan",
+      "message": "We need to copy the open-source IB driver!"
+    }
+  ]
```

Comment:
Generally: add new line at the end of the document

Reply:
> Generally: add new line at the end of the document

Fixed. Added new lines in new files.